lib/tasks/ci.rake

```diff
@@ -1,16 +1,29 @@
 require 'jettywrapper' unless Rails.env.production? || Rails.env.development?
 require 'rest_client'
 
-desc 'Run continuous integration suite (assuming jetty is not yet started)'
+desc 'Run continuous integration suite (tests, coverage, rubocop, docs)'
 task :ci do
-  Rake::Task['rubocop'].invoke
   unless Rails.env.test?
     system('bundle exec rake ci RAILS_ENV=test')
   else
+    system('bundle exec rspec spec --color')
```

Comment:
we already have a `.rspec` file for configuration so this can just be `bundle exec rspec` i think